lib/modules/dosomething/dosomething_signup/includes/dosomething_signup.inc

```diff
@@ -95,8 +95,8 @@ class SignupEntityController extends EntityAPIController {
     // If the entity uid doesnt belong to current user:
     if ($entity->uid != $user->uid) {
       // And current user can't edit any reportback:
-      if (!user_access('edit any signup')) {
-        watchdog('dosomething_signup', "Attempted uid override for @entity by User @uid", 
+      if (!user_access('edit any signup') && !drupal_is_cli()) {
```

Comment:
does this work? I would think it would fail out above when it checks the entity uid and the user? 
this should probably be it's own line, like [in the global module](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_global/dosomething_global.module#L24)